addons/website/views/snippets/snippets.xml

```diff
@@ -1160,6 +1161,18 @@
                 <t t-set="model">utm.source</t>
                 <t t-set="call_with">display_name</t>
             </t>
+            <we-select string="⌙ Users"
+                data-name="conditional_visibility_opt"
+                data-dependencies="visibility_conditional"
+                data-attribute-name="data-logged"
+                data-no-preview="true"
+                data-save-attribute="visibilityValueLogged"
+                data-attribute-rule="visibilityValueLoggedRule"
+                data-attribute-default-value="true or false">
+                <we-button data-select-values="true">Visible for Logged In</we-button>
+                <we-button data-select-values="false">Visible for Logged Out</we-button>
+                <we-button data-select-values="true or false">Visible for Everyone</we-button>
```

Comment:
I don't think any of the code related to this "or" stuff is needed. Visible for everyone is just like current master, you don't see a attribute with "true or false" value everywhere... that's just the default value.